src/managers/GoogleCalendarManager.ts

```diff
@@ -115,7 +139,7 @@ export default class {
                 this.meetingPingsSchema.getChannelID(calendarID),
               ) as TextChannel;
               channel.send({
-                content: `Meeting starting <t:${Math.trunc(startTime.toSeconds())}:R>! ${mentions}`,
+                content: `**${event.summary}** starting <t:${Math.trunc(startTime.toSeconds())}:R>! ${mentions}`,
```

Comment:
Incredibly small nitpick: We have a guard in the main message embed to default the event title to 'Untitled Event'

`.setTitle('🗓️ ' + (event.summary || 'Untitled Event'))`

Can we use the same defaulted title var in both spots? Otherwise, I think this is ready to merge at the meeting this Weds 🚀